project-backend/Models/JobController/GetJobBids/GetJobBidsResponseObject.cs

```diff
@@ -0,0 +1,30 @@
+﻿using project_backend.Models.Bid;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace project_backend.Models.JobController.GetJobBids
+{
+    public class GetJobBidsResponseObject
+    {
+        public List<JobBidDTO> Bids { get; set; }
+        public GetJobBidsResponseObject(IQueryable<BidDAO> bidsInput)
+        {
+            Bids = bidsInput.Select(b => new JobBidDTO
+            {
+                Id = b.Id,
+                JobId = b.JobId,
+                Sum = b.Sum,
+                WorkerId = b.WorkerId
+            }
+            ).ToList();
```

Comment:
Seemed cleaner to do the mapping here.

Reply:
It's not ok to have objects interact with IQueryable. Basically, only the providers and at most controllers should have to work with that interface, since it's provided by the ado framework, and it's the thing that actually runs queries on the db, and just as it would seem odd to run a query in an objects' constructor this seems weird for the same reasons.

Reply:
Transformed this into a List. Hope this is better, otherwise what collection should be used?

Reply:
I guess any collection would do as long as it's ordered and not of the queryable family :? List is fine, it would've been converted to a list anyways